packages/adapter-node/README.md

```diff
@@ -15,7 +15,11 @@ export default {
 		adapter: adapter({
 			// default options are shown
 			out: 'build',
-			precompress: false
+			precompress: false,
+			env: {
```

Comment:
maybe call it `listen` instead of env so it's more obvious whats happening and add default values here?